guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+            <td>Miguel operates a lightning node so he can receive payments at his business</td>
+            <td>Wagner has a mobile bitcoin wallet that supports lightning</td>
+         </tr>
+         <tr>
+            <td>What's their role in the lightning network?</td>
+            <td>Rachel acts more like a hub for routing other's lightning payments, and is rarely the origin or endpoint</td>
+            <td>Miguel is usually the endpoint for lightning payments</td>
+            <td>Wagner is sometimes the origin and sometimes the endpoint for lightning payments</td>
+         </tr>
+         <tr>
+            <td>Where does it run?</td>
+            <td>Runs on dedicated hardware, like a Raspberry Pi or rack-mounted server</td>
+            <td>Runs on a cloud service provider</td>
+            <td>Runs on Wagner's phone</td>
+         </tr>
+         <tr>
+            <td>When is it online?</td>
+            <td>Always online</td>
+            <td>Always online</td>
+            <td>Only online when Wagner opens the app</td>
+         </tr>
+         <tr>
+            <td>How many lightning channels?</td>
+            <td>Many lightning channels</td>
+            <td>Several lightning channels</td>
+            <td>Few lightning channels</td>
+         </tr>
+         <tr>
+            <td>How much liquidity?</td>
+            <td>Massive liquidity</td>
+            <td>Medium liquidity</td>
+            <td>Less liquidity</td>
+         </tr>
+         <tr>
+            <td>Where does the liquidity come from?</td>
+            <td>Liquidity is self-funded</td>
+            <td>Liquidity is usually sourced externally (from an LSP, for example)</td>
+            <td>Liquidity is usually sourced externally (from an LSP, for example)</td>
+         </tr>
+         <tr>
+            <td>How much do they know about lightning?</td>
+            <td>Rachel is a lightning node operator and is very cognizant of every decision she makes with her node</td>
+            <td>Miguel knows enough about operating a node to make his business function</td>
+            <td>Wagner is not even aware that his phone has a lightning node on it</td>
+         </tr>
+      </tbody>
+   </table>
+</div>
+
+Currently, most of the thinking in the Design Guide is focused on mobile wallet users such as Wagner. However, it is helpful to understand that routing nodes play a very important role in getting user funds across the lightning network.
+
+## Liquidity examples
+
+The following examples are designed to help you understand how liquidity works on the lightning network.
+
+Lori and Lamar, two routing node operators, open a channel together. Lori initiates the channel opening with 1,000,000 sats. This is the amount she makes available for transactions in the channel. Her initial outbound liquidity is the full amount of 1,000,000 sats, and her initial inbound liquidity is 0 sats (since Lamar has not committed any sats to the channel).
```

Comment:
A point that is missing is that Lamar can send free payments directly to Lamar with this, on first read it sounded like channels are 'only' for routing but they are also useful for P2P payments.

Reply:
Payment channels _can_ be used for P2P payments, but I purposefully excluded them from this page. I think that P2P payments are an edge-case. I do not think they constitute not the majority of payments on LN.

Take the following example:

> Alice likes coffee. Bob has a coffeeshop. Alice frequents Bob's coffeeshop, she opens a payment channel between her lightning node and Bob's lightning node. Since she stops in the coffeeshop everyday, she is able to route a payment directly to Bob for zero fees. After a month of buying coffees, Alice rebalances the channel or opens a new channel with Bob.

Sure, Alice _can_ do all of that. But I don't find it likely that the 99% of users are going to do this. And every time I try and give an example like that to a designer to help them understand lightning liquidity, they become too distracted with the story about the coffee shop to realize that I am just trying to paint a simplified picture of how a payment channel works. Thus, I want to separate out the activity of mobile nodes and routing nodes.

Maybe a better use case for P2P payments is something like:

> BobCorp is the preferred widget vendor for AliceCorp. The AliceCorp accounting department realizes that they are spending a lot on routing fees paying BobCorp invoices. So, after some well placed slack messages, they are able to collaborate with their IT department and the BobCorp IT department to open a direct payment channel to BobCorp. Now, they can pay invoices all the time with zero routing fees.

But I just think that kind of situation is not a huge part of what we're designing for at the moment.

So in this page, I'm specifically excluding any mention of P2P because I think it's so incredibly rare and is not what a designer needs to understand to design good lightning wallets.